include/dfrn.php

```diff
@@ -1229,9 +1240,10 @@ public static function deliver($owner,$contact,$atom, $dissolve = false) {
 
 
 			if ($dfrn_version >= 2.1) {
-				if (($contact['duplex'] && strlen($contact['pubkey']))
-						|| ($owner['page-flags'] == PAGE_COMMUNITY && strlen($contact['pubkey']))
-						|| ($contact['rel'] == CONTACT_IS_SHARING && strlen($contact['pubkey']))) {
+				if (($contact['duplex'] && strlen($contact['pubkey'])) {
```

Comment:
Why is this ```[``` here? This would surely create some syntax error on your machine. I guess that you don't actually work on your own code on your server?